### ISSUE TITLE
aws-sdk-core

### ISSUE THREAD

**Issue by PurrBiscuit** [aws/aws-codedeploy-agent#31]
using release `v1.0-1.751` with bundler to install gem dependencies causes this error when trying to start child process:

```
2015-09-23 14:54:00 ERROR [codedeploy-agent(1506)]: Plugin codedeploy could not be loaded: Unable to activate codedeploy-commands-1.0.0, because aws-sdk-core-2.1.23 conflicts with aws-sdk-core (= 2.1.2).
2015-09-23 14:54:00 ERROR [codedeploy-agent(1506)]: booting child: error during start or run: Gem::ConflictError - Unable to activate codedeploy-commands-1.0.0, because aws-sdk-core-2.1.23 conflicts with aws-sdk-core (= 2.1.2) - /usr/lib/ruby/2.2.0/rubygems/specification.rb:2112:in `raise_if_conflicts'
```

looks like that lock to 2.1.2 is coming from: https://github.com/aws/aws-codedeploy-agent/blob/v1.0-1.751/vendor/specifications/codedeploy-commands-1.0.0.gemspec#L24

but the gemspec used by the Gemfile uses a pessimistic lock at (~> 2.1.0): https://github.com/aws/aws-codedeploy-agent/blob/v1.0-1.751/codedeploy_agent-1.1.0.gemspec#L18

causing `aws-sdk-core 2.1.23` to be installed through bundler.

**Comment by suryanarayanan** [comment]
Hi,
We fixed this issue after v1.0-1.751 in https://github.com/aws/aws-codedeploy-agent/commit/2731b001e93df59ee6b0b5e0f74999e2edc63970#diff-dfc9a23ca4981cde09134ca95fc49968

The next release of the agent would contain the fix.
Thanks.

**Comment by PurrBiscuit** [comment]
nice, thank you :+1:
